Based on glibc-2.3.2/glibc-2.3.2-powerpc-as.patch

Fixes the following errors when building glibc for ppc7450:

/tmp/ccwlHdbl.s: Assembler messages:
/tmp/ccwlHdbl.s:73: Error: Unrecognized opcode: `stvx'
/tmp/ccwlHdbl.s:74: Error: Unrecognized opcode: `mfvrsave'
/tmp/ccwlHdbl.s:300: Error: Unrecognized opcode: `lvx'
/tmp/ccwlHdbl.s:302: Error: Unrecognized opcode: `mtvrsave'
make[2]: *** [/usr/local/src/crosstool-0.28-rc34/build/powerpc-7450-linux-gnu/gcc-3.2.3-glibc-2.2.5/build-glibc/libio/iopopen.o] Error 1
make[2]: Leaving directory `/home/local/src/crosstool-0.28-rc34/build/powerpc-7450-linux-gnu/gcc-3.2.3-glibc-2.2.5/glibc-2.2.5/libio'
make[1]: *** [libio/subdir_lib] Error 2
make[1]: Leaving directory `/home/local/src/crosstool-0.28-rc34/build/powerpc-7450-linux-gnu/gcc-3.2.3-glibc-2.2.5/glibc-2.2.5'
make: *** [all] Error 2

Contributed by Tom Warzeka <waz@quahog.npt.nuwc.navy.mil>

===================================================================
--- glibc-2.2.5/sysdeps/powerpc/Makefile~	2004-08-31 14:59:15.000000000 -0400
+++ glibc-2.2.5/sysdeps/powerpc/Makefile	2004-09-02 19:47:21.000000000 -0400
@@ -1,7 +1,6 @@
 # We always want to use the new mnemonic syntax even if we are on a RS6000
 # machine.
-+cflags += -mnew-mnemonics -Wa,-mppc -mpowerpc
-asm-CPPFLAGS += -Wa,-mppc
++cflags += -mnew-mnemonics
 
 ifeq ($(subdir),gmon)
 sysdep_routines += ppc-mcount
